src/components/common/accordion.tsx

```diff
@@ -1,26 +1,30 @@
 import {
-    Accordion,
-    AccordionSummary,
-    AccordionDetails,
-    Typography,
+	Accordion,
```

Comment:
It would seem we now have a tabs, space swap in this PR

Reply:
i don't understand